### PR TITLE
feat: Add Realtime Call-Up Demo

### PR DESCRIPTION
**Pull request by chyroc**
## Release Notes

- **New Features**
	- Introduced a new demo application for real-time voice calls, including a user-friendly interface and microphone permission handling.
	- Added a "Quick Start" section in the README to guide users through setup and usage.

- **Documentation**
	- Updated README with prerequisites, setup instructions, and a visual representation of the application interface.

- **Styles**
	- Comprehensive CSS styles added for various components, ensuring a responsive design and modern appearance.

- **Configuration**
	- New ESLint and Webpack configurations to enforce coding standards and optimize the build process.

- **Bug Fixes**
	- Enhanced error handling for microphone access and call initialization.

- **Tests**
	- Custom Jest transformers for handling styles and file imports, ensuring a smooth testing experience.

### CODE REVIEW

File: examples/realtime-call-up/src/CallUp.tsx [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+        voiceId: voice?.value,
+      });
+
+      realtimeAPIRef.current = new RealtimeClient({
+        accessToken,
+        baseURL: 'https://api.coze.cn',
+        botId: bot.bot_id,
+        voiceId: voice?.value,
+        debug: true,
+        allowPersonalAccessTokenInBrowser: true,
```
Comment (jackshen310) [inline examples/realtime-call-up/src/CallUp.tsx:139]: 如果不是用pat的token，则allowPersonalAccessTokenInBrowser 这个参数可以去掉

File: examples/realtime-call-up/src/CallUp.tsx [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+      }, 1000);
+      setTimerInterval(interval);
+    } else {
+      console.log('📞 通话已结束');
+      handleEndCall();
+    }
+  };
+
+  useEffect(() => {
+    const viewport = document.createElement('meta');
```
Comment (jackshen310) [inline examples/realtime-call-up/src/CallUp.tsx:214]: 这个是用来干啥的，支持移动端？

**Comment by jackshen310** [comment]
@jsongo @Tecvan-fe  @chyroc  开源代码有没强制要全部使用英文文案

File: examples/realtime-call-up/src/CallUp.tsx (hunk 2 above)
Comment (chyroc) [reply examples/realtime-call-up/src/CallUp.tsx:214]: 对

**Review by jackshen310 — APPROVED** [review]

**Review by Tecvan-fe — APPROVED** [review]

**Pull request merged**